css/default/_stars-list.scss

```diff
@@ -0,0 +1,59 @@
+@import "../base";
+@import "variables";
+
+@include block(stars-list) {
+  @include element(header) {
+    /* widget header */
+  }
+  @include element(body) {
+    /* wudget footer */
+  }
+  @include element(list) {
+    /* item list */
+  }
+  @include element(item) {
+    /* list item */
+    vertical-align: middle;
+
+    @include modifier(active) {
+      /* active list item */
+      font-weight: bold;
+    }
+  }
+  @include element(star) {
+    /* item star */
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+    background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACcAAAAmCAYAAABH/4KQAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAB1WlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNS40LjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOkNvbXByZXNzaW9uPjE8L3RpZmY6Q29tcHJlc3Npb24+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgICAgIDx0aWZmOlBob3RvbWV0cmljSW50ZXJwcmV0YXRpb24+MjwvdGlmZjpQaG90b21ldHJpY0ludGVycHJldGF0aW9uPgogICAgICA8L3JkZjpEZXNjcmlwdGlvbj4KICAgPC9yZGY6UkRGPgo8L3g6eG1wbWV0YT4KAtiABQAABF9JREFUWAnlWF1LFFEYfnXVdVfT1XFVlKgUjVoTRAwvQ+sP5D8oUCTqooigK+miiKiLIkQh/4H+gSi6lEQC06gsKkJR17XdbT9cbbTzOPPuHmdHZ3Z3kKAXxvPxvuc5zzznzHvOSvS/2PtR1xU8Tr1viVNAwInE1Hs63oQTuMVOgAADiiWT1IzHKfUcIyepJitY0Ls7Qo5Vq6wgwuOUeo6QY9X8DUR4YNyntfL7WzA5WbXWNiI8TqlXMLloVL0DXVgxuc6+/HQjKojc6nPf5USK2qEUFGNj9eBDDPfnWhZE7kss/AATyqoxAe7jGO7PpcybHKuGyWTVeHIn1MuL3MJIVS0rcqqF6WSXsnoYkx1xeE/RQW6A1fmLL4RSqdYEJc+rKp3YJmpCDuMx2Gu9PaLl5h5DmSKaniGKxTP9Hg8tlRItu1z0w0uet4rb/XU9uPMmMBLdyERptaIPTzy3UZUJqCmq2NohnzFYbjcKHdoCgpdX7s2upxJEiwtEK1lT748tK6awy01xmXgRn4MrQfXF/nAtX1UKVTyCZpXQ2FtDVOKxJmTE4TaI/kkKIX4RRXfFSRIWqgp1ZWU5ttHvuppeVpBkglClo5PDjqacn8uoC2LnhtWJ9AeBBjpBBUuA4H3G+8rpUkxiRgxzp5VjIkgRc6HwJNpHoaBMrFPxDTRcC08xlyxycBwVwcOIgUd6WdFgA/uuGqUf7eUVfYnFRk5bIXWAiPEgBmwY5pIV03pNlpUdKNefKX0LkdDk5ib5mhrFR8Lnp/hiMcGe5VoXg+YXNWLl5RQOVCsDdddDr3W0fYXpssoRMsEqkUp6O3RvrqT0+Ol5kUZEKrEihllMl1Umh7c6o/j7AAbQuKp7WTk0bdYxlokB8yDFeH5LcgisHw6+K3eTWIxMLkI9V+NTAljAtBpvixxAcK6ixElh13bFiSAbj2Us2WdWt02OD/w6/SzliQ8riwyxPJaxzAjJfbbIrY36u3a2xAYt04aCECa2Iga/MRYYwAKmTMSsbovcejJ2EYNxRWJjYqwOl7Kf+zgWPsZgTI43K239OwLXqT1gPlclJJ6YcxdcyImB5oyyUjjhlhMVHYwp+4x1W+Rw0cTABpHnjLawlMn07EPmx8MkuR8lMOBjTNlnrNsit5kiPhvS442kfNU0c6rSP4SAb7HgWDhCPYeRNMNMg+sVy8QwNjZYevLnuNjCRJe6iUAK+QqbGlZ5jD6f9irDxoSKk+VTIjQa+03tiMOHgFsOlvvlLHqIvh8fLBsaGheZxdwsyWGS2dXQK4BjMyPDwyqqaanN67thdmBrEdpf3HAWE+Gn8Yj22wNHIG6+eLnuBqXf+FLyWMtlXdtOCL00sKgAxDFWq7hu4XJKJO7ZFqaTn8JNeyOkPhYvl/5tomObHvqAtSTHXxVI+b2e+2dvJh+J7WxBKdutvQxN4AdVMJG8i5sOY2dHaz228lyLW3n40T9YrxE7CMpePzCABUx7I/7RqL/cOvWBxZSYFwAAAABJRU5ErkJggg==");
```

Comment:
Yes I wasn't sure but at the end I was thinking: "OK, I want it to be usable without custom CSS (like all our widgets)". But indeed, maybe we could go for UTF-8 stars instead of images -> you would then be able to easily override the color.